### PR TITLE
B 81 add saving files to congratulation

### PR DESCRIPTION
**Pull request by AlexKievDeveloper**
1. I took into account your comments and made the appropriate edits.
2. I changed the files state back(from 755 to 644(default files state)). Files not empty. Please ignore it.
3. All 94 tests works fine.
4. No conflicts with branch migrate_to_spring.

### CODE REVIEW

File: src/main/java/com/greetingcard/web/controller/UserController.java [hunk 1]
```diff
@@ -21,11 +21,10 @@
 @RequestMapping(value = "/api/v1/")
 public class UserController {
     private SecurityService securityService;
-
+    @Autowired
     private int maxInactiveInterval;
 
-    @Autowired
-    private ObjectMapper objectMapper;
+    private ObjectMapper objectMapper = new ObjectMapper();
```
Comment (Yuliia-Sh) [inline src/main/java/com/greetingcard/web/controller/UserController.java:28]: objectMapper is bean so can be Autowired

**Review by Yuliia-Sh — APPROVED** [review]

File: src/main/java/com/greetingcard/entity/LinkType.java [hunk 2]
```diff
@@ -2,18 +2,33 @@
...
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 @Slf4j
 public enum LinkType {
-    VIDEO("VIDEO", 1), PICTURE("PICTURE", 2), AUDIO("AUDIO", 3), PLAIN_LINK("PLAIN_LINK", 4);
+    VIDEO("VIDEO", 1), PICTURE("PICTURE", 2, "img","image/jpeg", "image/jpg", "image/png"),
+    AUDIO("AUDIO",  3, "audio","audio/mpeg"), PLAIN_LINK("PLAIN_LINK", 4);
```
Comment (Yuliia-Sh) [inline src/main/java/com/greetingcard/entity/LinkType.java:13]: we will not store plain links, wont we?

**Review by Yuliia-Sh — APPROVED** [review]

File: src/main/java/com/greetingcard/web/controller/UserController.java (hunk 1 above)
Comment (AlexKievDeveloper) [reply src/main/java/com/greetingcard/web/controller/UserController.java:28]: Thanks, I'll take that into account in the next change.

Comment (AlexKievDeveloper) [reply src/main/java/com/greetingcard/web/controller/UserController.java:28]: > objectMapper is bean so can be Autowired

Thanks, I'll take that into account in the next change.

**Review by RomanKlishch — APPROVED** [review]

**Review by OleksandrGubariev — APPROVED** [review]

File: src/main/java/com/greetingcard/entity/LinkType.java (hunk 2 above)
Comment (AlexKievDeveloper) [reply src/main/java/com/greetingcard/entity/LinkType.java:13]: > we will not store plain links, wont we?

Yes, you are right

**Pull request merged**